From: win7-support-bot <win7@cromite.org>
Date: Sun, 26 Jul 2026 00:00:00 +0000
Subject: WIN7: sandbox-policy / appcontainer

Files:
 sandbox/policy/win/sandbox_win.cc

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
diff --git a/sandbox/policy/win/sandbox_win.cc b/sandbox/policy/win/sandbox_win.cc
index fc71a3cb8dc38..7e0fa0d1b456d 100644
--- a/sandbox/policy/win/sandbox_win.cc
+++ b/sandbox/policy/win/sandbox_win.cc
@@ -941,11 +941,6 @@ sandbox::ResultCode SandboxWin::StartSandboxedProcess(
     return result;
   }
 
-  if (sandbox_type == Sandbox::kGpu) {
-    config->SetLockdownDefaultDacl();
-    config->AddRestrictingRandomSid();
-  }
-
   CHECK_EQ(AddGenericConfig(config), SBOX_ALL_OK);
 
   std::string appcontainer_id;
